rskj-core/src/main/java/co/rsk/net/discovery/PeerExplorer.java

```diff
@@ -392,6 +484,10 @@ private void removeConnections(List<PeerDiscoveryRequest> expiredRequests) {
     }
 
     private void removeConnection(Node node) {
+        logger.debug("removeConnection - Removing node: [{}], " +
+                "nodeAddress hostName: [{}], " +
+                "nodeAddress port: [{}]", node.getHexId(), node.getAddress().getHostName(), node.getAddress().getPort());
+
```

Comment:
```suggestion
        if (logger.isDebugEnabled()) {
            InetSocketAddress address = node.getAddress();
            logger.debug("removeConnection - Removing node: [{}], " +
                    "nodeAddress hostName: [{}], " +
                    "nodeAddress port: [{}]", node.getHexId(), address.getHostName(), address.getPort());
        }
```
`node.getAddress()` seems to be quite heavy

Reply:
Done